vochain/scrutinizer/indexertypes/types.go

```diff
@@ -12,6 +12,19 @@ import (
 	"go.vocdoni.io/proto/build/go/models"
 )
 
+const (
```

Comment:
These types should have a prefix and also a comment, else its hard to understand how to use them